<p>For a method having a single parameter, the types of its return value and its parameter should never be the same.</p>

<h2>Noncompliant Code Example</h2>
<pre>
CreditCardNumber = "4628431641210183" //non-compliant
</pre>

<h2>Compliant Solution</h2>
<pre>
CreditCardNumber = json.creditCard
</pre>
